appveyor.yml

```diff
@@ -1,8 +1,23 @@
 # Test against this version of Node.js
 environment:
   matrix:
-  - nodejs_version: "7"
-  - nodejs_version: "6"
+    - nodejs_version: "8"
+      ESLINT_VERSION: "4"
```

Comment:
I think the issue is that I'm still using the linux syntax... `npm install eslint@$ESLINT_VERSION` 🤦‍♂️